includes/services/PageManager.php

```diff
@@ -120,9 +123,32 @@ public function getById($id): ?array
         return $page;
     }
 
-    public function getRevisions($page)
+    public function getRevisions($pageTag, $limit = 10000)
+    {
+        return $this->dbService->loadAll("
+            SELECT id, time, user FROM {$this->dbService->prefixTable('pages')} 
+            WHERE tag = '{$this->dbService->escape($pageTag)}' 
+            ORDER BY time DESC
+            LIMIT {$limit}
+        ");
```

Comment:
```suggestion
        "), $pageTag);
```
pour prendre en compte les modifications de la PR #812